apps/condo/domains/user/utils/googleRecaptcha3.js

```diff
@@ -4,39 +4,56 @@ const captchaConfig = conf.GOOGLE_RECAPTCHA_CONFIG ? JSON.parse(conf.GOOGLE_RECA
 const CAPTCHA_SCORE_URL = captchaConfig.CAPTCHA_SCORE_URL ? captchaConfig.CAPTCHA_SCORE_URL : 'https://www.google.com/recaptcha/api/siteverify'
 const SERVER_KEY = captchaConfig.SERVER_KEY
 const { SAFE_CAPTCHA_SCORE, TROW_ERRORS_ON_LOW_CAPTCHA_SCORE } = require('@condo/domains/user/constants/common')
+const mobileCapthaCheck = conf.MOBILE_CAPTCHA_CHECK
 
 if (isEmpty(SERVER_KEY)) {
     console.error('Google reCaptcha not configured')
 }
+if (isEmpty(mobileCapthaCheck)){
+    console.warn('Mobile reCaptcha not configured')
+}
 
 const onCaptchaCheck = ({ success, challenge_ts, hostname, score, action }) => {
     console.log(
         (score < SAFE_CAPTCHA_SCORE) ? '\x1b[31m' : '\x1b[32m',
-        `Recaptcha: ${action} - [${score}]: ${success}`,         
-        challenge_ts, 
+        `Recaptcha: ${action} - [${score}]: ${success}`,
+        challenge_ts,
         hostname,
         '\x1b[0m'
     )
-} 
+}
 
-const captchaCheck = async (response, action = '') => {
+const captchaCheck = async (response, action = '', mobilePlatform = '') => {
     if (conf.NODE_ENV === 'test') {
         return { error: null }
     }
+    if (mobilePlatform) {
+        try {
+            let result = false
+            eval(mobileCapthaCheck)
+            if (result) {
+                return { error: null }
+            }
+        } catch (error) {
+            console.warn('[error] mobile captcha check failed', error)
```

Comment:
I find this message ambigious, we say error, but call warn

Reply:
It's debugging log to check mobile integration. I will need to trace requests. 
The real error will be thrown on next lines